.github/dependabot.yml

```diff
@@ -18,11 +18,10 @@ updates:
       prefix: "chore(deps): "
     # force the lockfile and package.json to be updated
     versioning-strategy: increase
-    # We update @aws-cdk/* libraries ourselves (via `script/update-aws-cdk`)
+    # We update AWS CDK libraries ourselves (via `script/update-aws-cdk`)
     # Dependabot wants to update them individually, but they should all be done at once.
     ignore:
-      - dependency-name: "aws-cdk"
-      - dependency-name: "@aws-cdk/*"
+      - dependency-name: "aws-cdk-lib"
```

Comment:
Now that this is a monolithic package, can we allow dependabot to update it for us?

If so, I think we could also remove our update script and [some CI stuff](https://github.com/guardian/cdk/blob/5b42e24646dbc176826fc16999dc821783225726/.github/workflows/ci.yaml#L20-L24) which aims to stop us from forgetting to keep this library up to date?

Reply:
In theory, yes we can get Dependabot to help here. In practice, I'm not sure if we'd want to? Dependabot runs every week, and bumps to `aws-cdk-lib` will incur a new release - do we want to create a new major release so frequently?

Reply:
I think you're right that weekly updates to this library would be too frequent.

I think that we have a few options here:

1. Keep things as they are but update the comment here which is now out of date. Suggestion for this:

```
# We update AWS CDK libraries ourselves (via `script/update-aws-cdk`) as this is a breaking change for users and it results in a new major version being released
```

2. Allow dependabot to update this library too; update the schedule so that dependabot checks all dependencies less frequently (e.g. fortnightly or monthly).

3. Create a dedicated update schedule for this library only (using [allow](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow)) - I'm not 100% sure if this would work but in theory it would allow us to continue checking/updating most dependencies weekly _and_ automatically update `aws-cdk-lib` less frequently.

Personally I'd prefer to remove the customised automation around this dependency (and I don't see any huge risk presented by option 2) but I'm happy to go with 1 if you just want to get this merged.

Reply:
> Personally I'd prefer to remove the customised automation around this dependency

+1

Let's decouple it from this (already massive) change and do 1. for now.